package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
May want to remove this package-lock.json if not needed

Reply:
The package-lock file was included by mistake.  It's been removed